core/include/kernel/pseudo_ta.h

```diff
@@ -38,11 +38,7 @@ struct pseudo_ta_head {
 	TEE_Result (*invoke_command_entry_point)(void *pSessionContext,
 			uint32_t nCommandID, uint32_t nParamTypes,
 			TEE_Param pParams[TEE_NUM_PARAMS]);
-#if defined(X86_64)
-} __aligned(0x80);
```

Comment:
Can we remove this alignment for struct initcall and struct ts_store_ops as well?

Reply:
Yes, GCC will not do padding on ARM. 
The alignment of ts_store_ops has been removed, patch has been updated. 
Panic and deadloop will be resulted in when removing the alignment of initcall. So it will take more time to debug if not onsite. Can we merge this patch first? I will add one more JIRA to track initcall alignment and debug it when back to office.

Reply:
OK, please record it in JIRA.